flyteplugins/go/tasks/plugins/k8s/kfoperators/pytorch/pytorch.go

```diff
@@ -145,16 +145,17 @@ func (p pytorchOperatorResourceHandler) BuildResource(ctx context.Context, taskC
 			"Invalid TaskSpecification, unsupported task template version [%v] key", taskTemplate.GetTaskTypeVersion())
 	}
 
-	if *workerReplicaSpec.Replicas <= 0 {
-		return nil, fmt.Errorf("number of workers must be greater than 0")
+	jobSpec := kubeflowv1.PyTorchJobSpec{}
```

Comment:
Could we add a small unit test?

Reply:
Hey @pingsutw, I've removed `TestBuildResourcePytorchV1WithZeroWorker` as it was intended to fail in the case corrected with this PR. Replaced it with the `TestBuildResourcePytorchV1WithDifferentWorkersNumber` test, thank you.

Reply:
One unit test is failing. otherwise, lgtm.

Reply:
Should be fixed now, thanks.